integration/run.sh

```
@@ -139,7 +182,7 @@ export SKIPLOGCLEAN=1
 
 FILTER_ARGS=""
 if [ ! -z "${TEST_PATTERN}" ]; then
-  FILTER_ARGS=" -k ${TEST_PATTERN}"
+  FILTER_ARGS=" -k \"${TEST_PATTERN}\""
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)